smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/middleware/MiddlewareExecutionGenerator.kt

```diff
@@ -58,18 +58,18 @@ class MiddlewareExecutionGenerator(
             }
             """.trimIndent()
         )
-        // Swift can't infer the generic arguments to `create` for some reason
```

Comment:
lol Swift can't infer the generic arguments to `create` because it has nothing to infer from, unless you explicitly give a type for the variable the return value will be assigned to.